process-engine/process_engine_runtime
# Support postgres and sqlite

## What did you change?

* added configs for both environments:
  * sqlite
  * postgres
* added error handling if an unavailable environment has been used

## How can others test the changes?

* install process_engine_runtime globally by running `npm i -g @process-engine/process_engine_runtime@feature~support_postgres_and_sqlite`
* run anywhere: `process-engine`
   * should start the process engine with `sqlite` environment (should work without postgres running)
* run anywhere: `NODE_ENV=sqlite process-engine`
   * should have the same effect as the former command
* run anywhere: `NODE_ENV=postgres process-engine`
   * should work with a running postgres instance
   * should fail to start without a running postgres instance
* run anywhere: `NODE_ENV=unknown process-engine`
   * should exit with exit code 1 and show an error: `
Configuration for environment "unknown" is not available.
Please make sure the configuration files are available at: {{YOUR_PATH}}/config/bla`

## PR-Checklist

Please check the boxes in this list after submitting your PR:
